service/controller/chart/v1/resource/release/create.go

```diff
@@ -29,7 +29,12 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 
 		tarballPath, err := r.helmClient.PullChartTarball(ctx, tarballURL)
 		if err != nil {
```

Comment:
Why is this still matching all errors? We said this does not make sense.

Reply:
I didn't see there are other two usage 🤦‍♂ Sorry.